cmd/up/up.go

```diff
@@ -1034,9 +1038,6 @@ func (up *upContext) shutdown() {
 	if up.Forwarder != nil {
 		up.Forwarder.Stop()
 	}
-	if err := config.DeleteStateFile(up.Dev); err != nil {
```

Comment:
Deleting the state file here can generate rc with parallel `exec` commands

Reply:
the state file was added so that the vscode plugin can give the users updates as the state of it changes. Not sure about the impact of this change, but keep that in comparison. It's better if the statefile is only modified during the lifecycle of "okteto up"/"okteto down".